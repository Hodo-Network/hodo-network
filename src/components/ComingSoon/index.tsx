import React from "react";

export default function ComingSoon() {
  return (
    <div className='bg-white overflow-hidden shadow rounded-lg'>
      <div className='px-4 py-5 sm:p-6'>
        <p className='text-lg font-medium'>Coming Soon</p>
      </div>
    </div>
  );
}
