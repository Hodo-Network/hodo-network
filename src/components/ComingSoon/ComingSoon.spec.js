import { cleanup } from "@testing-library/react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import ComingSoon from "./index";

afterEach(cleanup);

test("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<ComingSoon />, div);
});

test("matches snapshot", () => {
  const tree = renderer.create(<ComingSoon />).toJSON();
  expect(tree).toMatchSnapshot();
});
