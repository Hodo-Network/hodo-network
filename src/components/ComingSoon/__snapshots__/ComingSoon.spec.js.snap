// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="overflow-hidden"
>
  <div
    className="px-4 py-5 sm:p-6"
  >
    <p
      className="text-xl font-medium dark:text-white"
    >
      Coming Soon
    </p>
  </div>
</div>
`;
